### PR TITLE
Add testimonials to landing page

### PR DESCRIPTION
**Pull request by dtuite**
Preview: https://deploy-preview-757--roadie.netlify.app/

### CODE REVIEW

File: src/components/landing/Testimonials/Avatar.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <img
+        src={avatar.jpeg || avatar.png}
+        alt={avatar.alt}
+        className="h-12 w-12 rounded-full"
+        height="100"
+        width="100"
+      />
+    </picture>
+  </div>
+);
```
Comment (dtuite) [inline src/components/landing/Testimonials/Avatar.js:24]: Extracted this from the existing testimonial component

File: src/components/landing/Testimonials/SidebarTestimonial.js [hunk 2]
```diff
@@ -33,21 +35,13 @@ const Testimonial = () => (
...
-              height="100"
-              width="100"
-            />
-          </picture>
-          </div>
+          <Avatar
+            avatar={{
+              webp: enriqueAvatar,
+              png: enriqueAvatarPng,
+              alt: `The fact of a man with a slight beard and glasses. He is looking up and to the right slightly.`,
```
Comment (Xantier) [inline src/components/landing/Testimonials/SidebarTestimonial.js:42]:
```suggestion
              alt: `The face of a man with a slight beard and glasses. He is looking up and to the right slightly.`,
```

Too philosophical with 'fact'

File: src/components/landing/Testimonials/TestimonialsCloud.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+import jonAvatar from '../../../../content/assets/home/testimonial/jon-ncsa/avatar.webp';
+import jonAvatarJpeg from '../../../../content/assets/home/testimonial/jon-ncsa/avatar.jpeg';
+
+const TESTIMONIALS = [{
+  title: 'Incredible',
+  text: (
+    <>
+      <p className="prose mb-2">The Roadie team have been incredible to work with. They’re obviously Backstage experts, and their platform provides us with a ton of flexibility and integrations.</p>
+      <p className="prose">We&apos;ve been able to start using Backstage must faster and we don’t have to worry about the maintenance.</p>
```
Comment (Xantier) [inline src/components/landing/Testimonials/TestimonialsCloud.js:19]: Using `&apos;` here but not above or later
```suggestion
      <p className="prose">We've been able to start using Backstage must faster and we don’t have to worry about the maintenance.</p>
```

File: src/components/landing/Testimonials/TestimonialsCloud.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      <p className="prose">The Roadie team has been incredibly proactive in communicating updates and responsive to all our questions.</p>
+    </>
+  ),
+  author: {
+    name: 'Jon Stern',
+    role: 'Director, NCSA',
+    avatar: {
+      webp: jonAvatar,
+      jpeg: jonAvatarJpeg,
+      alt: `A sketch of some carachters playing an early form of football`,
```
Comment (Xantier) [inline src/components/landing/Testimonials/TestimonialsCloud.js:62]:
```suggestion
      alt: `A sketch of some characters playing an early form of football`,
```

**Review by Xantier — APPROVED** [review]

File: src/components/landing/Testimonials/TestimonialsCloud.js (hunk 3 above)
Comment (dtuite) [reply src/components/landing/Testimonials/TestimonialsCloud.js:19]: Linter complains if I do that.

Comment (Xantier) [reply src/components/landing/Testimonials/TestimonialsCloud.js:19]: That seems weird how it complains on one thing but not the second one

**Review by Xantier — DISMISSED** [review]

File: src/components/landing/Testimonials/TestimonialsCloud.js (hunk 3 above)
Comment (dtuite) [reply src/components/landing/Testimonials/TestimonialsCloud.js]: It makes a bit more sense now hopefully.

**Comment by dtuite** [comment]
Merging this so I can work off it. It was essentially already reviewed.

**Pull request merged**